pip_audit/_dependency_source/requirement.py

```diff
@@ -89,6 +89,9 @@ def collect(self) -> Iterator[Dependency]:
                 reqs: List[InstallRequirement] = []
                 req_names: Set[str] = set()
                 for req in rf.requirements:
+                    if req.is_editable:
+                        # We can't evaluate editable requirements
+                        continue
```

Comment:
We have a concept of "skipped" dependencies that end up in the output to signal to users that certain dependencies couldn't be audited. I think we should be yielding this here instead of just ignoring it.

Reply:
SGTM, feel free to push to this branch.